gradle.properties

```diff
@@ -1,8 +1,8 @@
 org.gradle.daemon=false
 org.gradle.parallel=false
 org.gradle.jvmargs=-Xms512m -Xmx512m
-scalaVersion=2.13.6
-kafkaVersion=3.1.0
+scalaVersion=2.13.8
```

Comment:
I think https://github.com/linkedin/cruise-control/blob/91342ebfd5384bd0c402fc416d7875790581ecc2/kafka-cruise-control-start.sh#L33 needs to be updated as well

Reply:
Good point!